.github/workflows/sdks.yml

```diff
@@ -480,3 +480,115 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+  assemble-sdks:
+    runs-on: ubuntu-22.04
+    needs: [build-sdk-and-tests, get-latest-toolchain]
```

Comment:
Instead of creating a whole new job, why not just add these as steps to the linux `build-sdk-and-tests` jobs after the emulator test run? That way, you only need four of these steps, ie getting the arm cached SDKs and putting them together, though you will need an `if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}` for each.

Since the emulator runs always take the longest, that should make sure there's time for all the SDKs to be cached by then.

Reply:
OK, I've condensed the new job as suggested in 6b534f2.